src/mixins/default_controls.js

```diff
@@ -7,108 +7,127 @@
       scalingYOrSkewingX = controlsUtils.scalingYOrSkewingX,
       scalingXOrSkewingY = controlsUtils.scalingXOrSkewingY,
       scaleOrSkewActionName = controlsUtils.scaleOrSkewActionName,
-      objectControls = fabric.Object.prototype.controls;
+      rotationWithSnapping = controlsUtils.rotationWithSnapping,
+      rotationStyleHandler = controlsUtils.rotationStyleHandler,
+      changeWidth = controlsUtils.changeWidth;
 
-  objectControls.ml = new fabric.Control({
-    x: -0.5,
-    y: 0,
-    cursorStyleHandler: scaleSkewStyleHandler,
-    actionHandler: scalingXOrSkewingY,
-    getActionName: scaleOrSkewActionName,
-  });
+  class ObjectControls {
```

Comment:
since we are not migrating yet, this should still be the function + prototype class.